src/NavBar/NavBar.scss

```diff
@@ -125,20 +124,28 @@
   }
 
   //no Hamburger for tablets and larger
-
-  @media (min-width: 768px) {
-
-    .menu {
-      font-size: 22px;
-      padding-right: 10px;
-      max-height: none;
-    }
-
-    .menuIcon {
-      display: none;
+  
+      @media (min-width: 768px) {
+      .menuIcon {
+        display: none;
+      }
+      
+      .menuButton:checked ~ .menu {
+        background-color: transparent;
+        position: unset;
+        top: unset;
+        right: unset;
+        width: unset;
+        max-height: unset;
```

Comment:
can this just be `display: none`?
I think ideally we just don't want any of this stuff to appear for desktop sizes